cmd/kots/cli/install.go

```diff
@@ -166,6 +166,7 @@ func InstallCmd() *cobra.Command {
 				ApplicationMetadata:       applicationMetadata,
 				License:                   license,
 				ConfigValues:              configValues,
+				Airgap:                    v.GetBool("airgap"),
```

Comment:
There is already a variable `IsAirgap` defined above that also takes under consideration the `airgap-bundle` flag, and it would be a good idea to rename `Airgap` field in the DeployOptions to `IsAirgap`
```suggestion
				Airgap:                    isAirgap,
```